app/models/topic_tracking_state.rb

```diff
@@ -189,7 +189,7 @@ def self.publish_unread(post)
         payload: payload
       }
 
-      MessageBus.publish(self.unread_channel_key(tu.user_id), message.as_json, group_ids: group_ids)
+      MessageBus.publish(self.channel, message.as_json, group_ids: group_ids)
```

Comment:
It doesn't seem right to me that we seem to have lost the user for the unread channel, shouldn't that still be there?